pkg/crc/preflight/preflight_common.go

```diff
@@ -23,7 +23,7 @@ func preflightCheckSucceedsOrFails(configuredToSkip bool, check PreflightCheckFi
 		return
 	}
 
-	log.Fatal(err.Error())
+	log.FatalF(" Above Check Failed: %+v", err)
```

Comment:
:+1:

Reply:
So our error should be correctly returned I think current solution doesn't going well in long term so better fix the function which return that error as `nil` even result is false.